mixed_pond_experimental/sensitivity_setup.py

```diff
@@ -0,0 +1,222 @@
+
+    W_2 = W_z * 3.6
+    air_temp_line = read_air_temp(day_argue)
+    T_ak = float(air_temp_line['avg_temp']) #kelvin
+    print(T_ak)
+    T_ac = T_ak -273.15 #degree celcius
+
+    # e_s, saturated vapor pressure needs to be in T_wc deg celcius
+
+    T_wc = T_wk - 273.15
+    e_s = 25.374 * math.exp(17.62 - 5271/T_wc)
+
+    RH = Rh
+
+    # e_a, water vapor pressure above the pond surface; unit mmHg
+    
+    e_a = RH * 25.374 * math.exp(17.62 - 5271/T_ac)     
+
+    phi_e = N* W_2 * (e_s- e_a)
+    return(phi_e)
+
+#create function for phi_c sensible heat transfer
+
+def calculate_phi_c(T_wk, day_argue):
+   
+    air_temp_line = read_air_temp(day_argue)
+    print(air_temp_line)
+    T_ak = float(air_temp_line['avg_temp']) #kelvin
+    print(T_ak)
+    T_wc = T_wk - 273.15 #convert to deg celcius
+    T_ac = T_ak - 273.15 
+
+    phi_c = 1.5701 * W_z * (T_wc-T_ac)
+
+    return(phi_c)
+
+############### Creating simulation loop for daily values #############################
+# Energy Flux equation: phi_net = phi_sn + phi_at - phi_ws - phi_e - phi_c   
+# Heat at t-1 time step: H_t_1 = T_wk * volume * water_heat_capacity * water_density
+# Heat at t: H_t = H_t_1 + (phi_net * area * t)
+# T_w = T_wc + H_t/ (volume * water_heat_capacity * water_density)
+
+
+# loop for energy flux equation 
+def main_simulation_loop():
+    
+    global T_wk
+
+    count = 0
+    for day_argue in list(range(1, 1096)):
+        
+        
+        count = count + 1
+
+        print(f"Iteration {count} start - T_wk: {T_wk}")
+
+        phi_sn = calculate_phi_sn(day_argue)
+        print(f'phi_sn: {phi_sn}')
+        phi_at = calculate_phi_at(day_argue)
+        print(f'phi_at: {phi_at}')
+        phi_ws = calculate_phi_ws(T_wk, day_argue)
+        print(f'phi_ws: {phi_ws}')  
+        phi_e = calculate_phi_e(T_wk, day_argue)
+        print(f'phi_e: {phi_e}')
+        phi_c = calculate_phi_c(T_wk, day_argue)
+        print(f'phi_c: {phi_c}')
+        
+        phi_net = phi_sn + phi_at - phi_ws - phi_e - phi_c 
+
+        print(f'iteration: {count}, phi_net: {phi_net}')
+        
+        T_wC = T_wk - 273.15 #change to degree celcius   
+
+
+        H_t_1 = T_wC * volume * water_heat_capacity * water_density
+        #check if K or C
+        print(f'iteration: {count}, H_t_1: {H_t_1}')
+        H_t = H_t_1 + (phi_net * area * t)
+
+        
+        #T_wC_o = 26.5
+        
+        T_w =  H_t/ (volume * water_heat_capacity * water_density)
+        print(f'iteration: {count}, T_w: {T_w}')
+
+        #add T_w to a list somehow
+        T_wC_vec.append(T_w)
+
+        T_wk = T_w + 273.15 #convert back to kelvin
+        print(T_wk)
+        
+    
+    print(T_wC_vec)
+    
+    T_wC = np.array(T_wC_vec)
+
+    df = pd.DataFrame(T_wC)
+    
+    df1= pd.concat([air_temp_data, df], axis = 1)
+    
+    df1.to_csv('Water_temp_daily.csv',index=False)
+
+    plt.plot(T_wC)
+    plt.plot(data['degC_avg'])
+    plt.plot(watertemp['day_avg'])
+    plt.show()
```

Comment:
Can you add the legend with labels to the plot?

`plt.plot(watertemp['day_avg'], label="observed water temp")`
`plt.legend()`
`plt.show()`